kolibri/core/assets/src/vue/core-base.vue

```diff
@@ -60,10 +72,37 @@
       title(newVal, oldVal) {
         document.title = `${newVal} - Kolibri`;
       },
+      'windowSize.breakpoint': function (newVal, oldVal) {
```

Comment:
this is throwing a lint error - `Expected method shorthand         object-shorthand`